src/Core/Users/Actions/ResetPassword.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace GetCandy\Api\Core\Users\Actions;
+
+use Illuminate\Support\Facades\Hash;
+use Lorisleiva\Actions\Action;
+
+class ResetPassword extends Action
+{
+    /**
+     * Determine if the user is authorized to make this action.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the action.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'current_password' => 'string|min:6',
```

Comment:
Do we need to be worried if the password meets the min? What if the user was imported from another system?

Reply:
For current password no just needs to conform for new password